src/components/button/BackToTopButton.tsx

```diff
@@ -0,0 +1,57 @@
+import styled, { keyframes } from "styled-components";
+import { Button } from "./Button";
+import { Icon } from "components/icon";
+import { faChevronUp } from "@fortawesome/pro-solid-svg-icons";
+import { useState, useEffect } from "react";
+import theme from "theme";
+import { withHover } from "styles/mixins";
+import { fadeIn } from "styles/animations";
+
+const ScrollButton = styled(Button)`
+  position: fixed;
+  right: 1rem;
+  bottom: 1rem;
+  padding: 1rem;
+
+  display: ${(props) => props.isVisible ? "" : "none"};
+  animation: ${fadeIn} 200ms both;
+  
+  ${withHover`
+    background-color: ${theme.colors["solid-on-card"]}; 
+    color: ${theme.colors["text-primary"]}; 
+  `}
+`;
+
+export function BackToTopButton() {
+  const [isButtonVisible, setIsButtonVisible] = useState(false);
+  
+  useEffect(() => {
+    window.addEventListener("scroll", () => {
+      if (window.scrollY > 2000) {
+        setIsButtonVisible(true);
+      } else {
+        setIsButtonVisible(false);
+      }
+    });
+  }, []);
+
+  const scrollUp = () => {
+        
+    window.scrollTo({
+      top: 0,
+      behavior: "auto",
```

Comment:
I'm still unsure as well😅

I decided to go with ``auto`` for now because it feels snappy, which I feel somewhat falls in line with the rest of the client. However, I can see ``smooth`` being a better fit for mobile. I'm open to both options.